integration-tests/README.md

```diff
@@ -117,6 +117,33 @@ will be fed into config
 properties `quarkus.azure.app.configuration.endpoint` / `quarkus.azure.app.configuration.id` / `quarkus.azure.app.configuration.secret`
 of `azure-app-configuration` extension in order to set up the connection to the Azure App Configuration store.
 
+### Creating Azure Key Vault
+
+Run the following commands to create an Azure Key Vault, set permission and export its connection string as an environment
+variable.
+
```

Comment:
Mention that the steps listed here are also performed by the GitHub Actions workflows via the `build-with-native.sh` file.

Reply:
Good catch, see https://github.com/quarkiverse/quarkus-azure-services/commit/5fce42ce138dd83c3e4baf06b1a1b76ec8c82829.